manifests/broker.pp

```diff
@@ -15,7 +15,7 @@
     Service[$broker_service] -> Exec['migrate_pulp_db']
   } else {
     if $pulp::messaging_transport == 'qpid' {
-      package { 'qpid-tools': ensure => installed } -> Class['pulp::service']
+      Class['qpid::tools'] -> Class['pulp::service']
```

Comment:
Right but `Class['qpid::tools']` isn't including the class, it's just forcing the dependency if it's present.

I think you need to do `include qpid:tools` before this line to account for the case you put qpid on a different server, which doesn't apply to us but could for others.